doc/content/getting-started/troubleshooting/_index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Troubleshooting"
+description: ""
+---
+
+This section provides help for common issues you may encounter when getting started with {{% tts %}}.
+
+<!--more-->
+
+## My gateway won't connect. What do I do?
+
+- Double check your gateway settings in {{% tts %}}. Ensure your Gateway EUI is correct
+- Double check your configuration settings on your gateway. Is the network address the same address you use to connect to {{% tts %}}? Are your ports correct?
+- If using CUPS, double check your API Keys and certificates
+
+## My device won't join. What do I do?
+
+- Double check your Device EUI, Join EUI, Keys, and Frequency Plan settings
+- Double check your network connection. If 
+there is a slow connection from the server to the gateway, the join accept message may be sent too late (this can happen when a gateway uses 3G as a backhaul)
+- Check for overused App Nonce
+
+## No downlinks are reaching my device
+
+- Check duty cycle restrictions
+- Device clock drift often occurs when SF12 is used
+- Check antenna connections
+
+## How do I get gateway logs?
+
+Start recording gateway gateway logs using the following command:
+
+```bash
+$ ttn-lw-cli events subscribe --gateway-id <gateway ID>
+```
+
```

Comment:
Users can also see these events from the gateway page on the Console, worth including here